roxctl/common/auth/token.go

```diff
@@ -45,14 +45,21 @@ func (t tokenMethod) retrieveAuthToken() (string, error) {
 			return "", errors.Wrapf(err, "could not read token from %q", tokenFile)
 		}
 		apiToken = token
+	} else if tokenFileFromEnv := env.TokenFileEnv.Setting(); tokenFileFromEnv != "" {
```

Comment:
Wondering if we should make this a switch case now instead of chaining the if / else here for readability; would you mind making the change?

Reply:
Switch does not allow setting variables in the case part.
Now stepping back and checking the behaviour of `flags.APITokenFile()`, the extra if statement is not required.
I'll add a series of tests to ensure the precedence logic of the code block is the expected one:
- command line flag
- file env variable
- token env variable